layouts/events/list.html

```diff
@@ -123,7 +149,118 @@ <h5 class="modal-title" id="suggestionModalLabel">Events & training suggestion</
 </section>
 
 <script>
- // Functions need to be run when the DOM is ready
+  // Initialise session tracking for anti-spam
+  var sessionStartTime = Date.now();
+  var sessionId = generateSessionId('events');
+
+  function submitEventsForm() {
+    // Validate form-specific fields
+    if (!validateEventsForm()) {
```

Comment:
Same as above, code specific to this file only used once, so no need of function ?